content/hardware/05.pro-solutions/solutions-and-kits/wisgate-edge-pro/datasheet/datasheet.md

```diff
@@ -0,0 +1,490 @@
+| EC25-J (Japan Region)          | LTE FDD: B1/B3/B8/B18/B19/B26 <br> LTE TDD: B41 <br> WCDMA: B1/B6/B8/B19                               |
+| EC25-AU (Brazil Region)        | LTE-FDD: B1/B2/B3/B4/B5/B7/B8/B28 <br> LTE-TDD: B40 <br> WCDMA: B1/B2/B5/B8 <br> GSM/EDGE: B2/B3/B5/B8 |
+
+<div style="page-break-after: always;"></div>
+
+## Rating
+
+<p style="text-align: justify;">
+The following section provides electrical and environmental parameters for the safe and reliable operation of the WisGate Edge Pro. Always use the device within these specified conditions to ensure longevity and optimal performance.
+</p>
+
+### Recommended Operating Conditions
+
+| **Description**           | **Value**                     |
+|---------------------------|-------------------------------|
+| Operating Temperature     | -30 °C to +55 °C              |
+| Storage Temperature       | -40 °C to +85 °C              |
+| Operating Humidity        | 0% to 95% (non-condensing)    |
+| Storage Humidity          | 0% to 95% (non-condensing)    |
+| Ingress Protection Rating | IP67 (dust-tight, waterproof) |
+| Pollution Degree          | 2 (IEC 61010 compliant)       |
+
+<div style="background-color: #FFFFE0; border-left: 6px solid #FFD700; margin: 20px 0; padding: 15px;">
+<p style="text-align: justify;">
+<strong>Note:</strong> The device's IP67 rating ensures it is fully dust-protected and can withstand immersion up to defined limits, suitable for harsh outdoor environments.
+</p>
+</div>
+
+### Power Specification
+
+<p style="text-align: justify;">
+The WisGate Edge Pro gateway supports multiple power supply options suitable for various installation scenarios:
+</p>
+
+- **Power over Ethernet (PoE)**: Supports IEEE 802.3af standard (37 to 57 VDC input voltage). Use an IEEE 802.3af compliant PoE injector or switch for proper operation. The PoE port provides 10/100 Mbps Ethernet connectivity.
+- **External DC Power Supply**: Supports a dedicated external 12 VDC power source via the DC barrel jack. Use the 12V/1A power adapter or a regulated 12 VDC supply with at least 2 A capacity to ensure stable operation.
+
+| **Property**                | **Min** | **Typical** | **Max** | **Unit** |
+|-----------------------------|---------|-------------|---------|----------|
+| PoE Input Voltage (802.3af) | 37      | -           | 57      | VDC      |
+| DC Input Voltage            | 11      | 12          | 13      | VDC      |
+| Power Consumption           | -       | 12          | 17      | W        |
+
+<div style="background-color: #FFFFE0; border-left: 6px solid #FFD700; margin: 20px 0; padding: 15px;">
+<p style="text-align: justify;">
+<strong>Warning:</strong> Always use IEEE 802.3af compliant PoE injectors or switches to prevent potential damage or malfunction. If using a DC power supply, ensure it is a regulated 12V source capable of supplying at least 2A to accommodate peak power consumption.
```

Comment:
```suggestion
<strong>Warning:</strong> Always use IEEE 802.3af compliant PoE injectors or switches to prevent potential damage or malfunction. If using a DC power supply, ensure it is a regulated 12 VDC source capable of supplying at least 2 A to accommodate peak power consumption.
```